### PR TITLE
Added a command to show spawned hosts

### PR DESCRIPTION
**Pull request by Lex-2008**
Ticket: CFE-3740

### CODE REVIEW

**Comment by olehermanse** [comment]
@Lex-2008 @vpodzime How about we make it just:

```
cf-remote show
```

or

```
cf-remote list
```

I was thinking we can make it possible to name / group hosts which are not spawned by cf-remote as well, and they would all show up there.

File: cf_remote/commands.py [hunk 1]
```diff
@@ -484,6 +485,25 @@ def init_cloud_config():
...
 
+def show_spawned():
+    if not os.path.exists(CLOUD_STATE_FPATH):
+        print("No saved cloud state info")
+        return 1
+
+    vms_info = read_json(CLOUD_STATE_FPATH)
+    vms = []
+    for group_name in vms_info:
+        vms += [{'platform': group_name}]
```
Comment (vpodzime) [inline cf_remote/commands.py:496]: The group name doesn't generally have to be the platform, it can be an arbitrary name.

**Review by vpodzime — COMMENTED** [review]
Some example output would be nice for the review process.

**Review by olehermanse — CHANGES_REQUESTED** [review]
@Lex-2008 please change this to be a new command:

```
cf-remote list
```

**Comment by Lex-2008** [comment]
@olehermanse : there already is a `cf-remote list` command, to "List CFEngine packages available for download":
```
[lex@flower2 cf-remote]$ python -m cf_remote list
Available releases: master, 3.19.0, 3.18.x, 3.18.1, 3.18.0, 3.15.x, 3.15.5, 3.15.4, 3.15.3, 3.15.2, 3.15.1, 3.15.0, 3.15.0b1
Using 3.18.1 LTS (default):
https://cfengine-package-repos.s3.amazonaws.com/enterprise/Enterprise-3.18.1/hub/debian_9_x86_64/cfengine-nova-hub_3.18.1-1.debian9_amd64.deb
https://cfengine-package-repos.s3.amazonaws.com/enterprise/Enterprise-3.18.1/hub/debian_10_x86_64/cfengine-nova-hub_3.18.1-1.debian10_amd64.deb
...
```

I can make this one into `cf-remote list-spawned`, then, maybe?

Also there is `cf_remote spawn --ansible-inventory` which is similar to this one, only it produces ansible-readable output, instead of human/awk/grep-readable one:

```
$ python -m cf_remote spawn --ansible-inventory
[r8-hub]
vpr8-hub-rhel-8-x64hub0 ansible_host=3.249.70.93 ansible_user=ec2-user ansible_ssh_extra_args="-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"

[d10-clients]
vpd10-clients-debian-10-x64client0 ansible_host=34.243.193.4 ansible_user=admin ansible_ssh_extra_args="-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
vpd10-clients-debian-10-x64client1 ansible_host=34.243.91.20 ansible_user=admin ansible_ssh_extra_args="-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"

...
```

**Comment by vpodzime** [comment]
> Also there is `cf-remote spawn --ansible-inventory` which is similar to this one, only it produces ansible-readable output, instead of human/awk/grep-readable one:

That should be moved to the new command.

File: cf_remote/main.py [hunk 2]
```diff
@@ -106,6 +105,9 @@ def get_args():
                     action="store_true")
     # TODO: --region (optional)
 
+    sp = subp.add_parser("list-spawned", help="List already spawned hosts")
```
Comment (olehermanse) [inline cf_remote/main.py:108]: Ah sorry, I forgot about list.

list-spawned is too long and we just talked about how this should not be related to "spawned" hosts.

Please change it to:

```suggestion
    sp = subp.add_parser("show", help="List already spawned hosts")
```

File: cf_remote/main.py [hunk 3]
```diff
@@ -106,6 +105,9 @@ def get_args():
...
+    sp = subp.add_parser("ansible-inventory", help="Print Ansible inventory with spawned hosts")
```
Comment (olehermanse) [inline cf_remote/main.py:109]: This should be an option to the other command;

```
cf-remote show --ansible-inventory
```

**Review by olehermanse — CHANGES_REQUESTED** [review]

File: cf_remote/main.py (hunk 2 above)
Comment (olehermanse) [reply cf_remote/main.py]: Fixed myself.

File: cf_remote/main.py (hunk 3 above)
Comment (olehermanse) [reply cf_remote/main.py]: Also fixed this.

File: cf_remote/commands.py (hunk 1 above)
Comment (olehermanse) [reply cf_remote/commands.py]: I've changed / fixed this.

**Comment by olehermanse** [comment]
@Lex-2008 I finished this as I need it to implement something else. Feel free to review my changes after merge.

**Pull request merged**